test/GitcoinGovernor.t.sol

```diff
@@ -236,3 +240,190 @@ contract GitcoinGovernorProposalTest is GitcoinGovernorTest {
     assertEq(timelock.admin(), address(governor));
   }
 }
+
+contract GitcoinGovernorAlphaPostProposalTest is GitcoinGovernorProposalTestHelper {
+  address constant USDC_ADDRESS = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+  address constant RAD_ADDRESS = 0x31c8EAcBFFdD875c74b94b077895Bd78CF1E64A3;
+
+  IERC20 usdcToken = IERC20(USDC_ADDRESS);
+  IERC20 radToken = IERC20(RAD_ADDRESS);
+
+  function setUp() public override {
+    super.setUp();
```

Comment:
Wdyt about making this more explicit? I think avoiding `super` is a good habit to get into, since it's not intuitive what it refers to if you're inheriting from multiple contracts (but you can still use it, good related tweet thread [here](https://twitter.com/cleanunicorn/status/1565672158516236288)
```suggestion
    GitcoinGovernorProposalTestHelper.setUp();
```

Reply:
Done in db5391726ec0eaec9bdabc998cdd5beba65d4967